grading-vm/images/testing/echo-server/index.js

```diff
@@ -0,0 +1,33 @@
+import express, { json } from "express";
+
+const PORT = 9001;
+
+const app = express();
+app.use(json({ limit: "1mb" }));
+
+const responses = {};
+
+app.get("/", (_, res) => {
+  res.send("Hello world!");
+});
+
+app.get("/job-output/:key", (req, res) => {
```

Comment:
For debugging purposes (since that's all this is!), add `app.get("/job-output", (req, res) => { return res.json(Objects.keys(responses)); })` so we can see what responses have been received at all.